pkg/payment/payment.go

```diff
@@ -12,11 +12,10 @@ import (
 )
 
 const (
-	baseURL   = "https://api.mercadopago.com/v1/"
-	postURL   = baseURL + "payments"
-	searchURL = baseURL + "payments/search"
-	getURL    = baseURL + "payments/{id}"
-	putURL    = baseURL + "payments/{id}"
+	urlBase             = "https://api.mercadopago.com/v1/"
+	urlPayment          = urlBase + "payments"
+	urlPaymentSearch    = urlBase + "payments/search"
+	urlPaymentWithParam = urlBase + "payments/{id}"
```

Comment:
imo fica melhor assim
```suggestion
	domain             = "https://api.mercadopago.com/v1"
	urlPayment          = domain + "/payments"
	urlPaymentSearch    = domain + "/payments/search"
	urlPaymentWithParam = domain + "/payments/{id}"
```